<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTreeWidget" name="treeWidget">
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionImport_Siege"/>
    <addaction name="actionExport_Siege"/>
   </widget>
   <addaction name="menuFichier"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionImport_Siege">
   <property name="text">
    <string>Import Siege...</string>
   </property>
  </action>
  <action name="actionExport_Siege">
   <property name="text">
    <string>Export Siege...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
